roles/sat6repro/tasks/main.yml

```diff
@@ -78,8 +78,12 @@
 - name: untar config files
   command: tar --selinux --overwrite -xvf /tmp/config_files.tar.gz  -C /
   when: perform_restore
+- name: Identify the default dhcp interface
+  shell: route | grep -m 1 '^default' | awk '{print $8}'
```

Comment:
I don't know if we want to assume that the default route interface is the one used for DHCP. This seems dangerous.